### PR TITLE
Automatic login for voting

### PR DESCRIPTION
**Pull request by johannth**
Appends a token=<token> query parameter to the link that we send to the reviewers.

I decide to plug into the current authentication scheme that uses JWT. 

The flow is as follow:

1. Generate a token that lasts for 15 days and passes along in the email.
2. When the server detects the token parameter and no cookie it will add the cookie (but only for 15 minutes).

Both the token and cookie are relatively short lived. If we would want to revoke them it would be easiest to just change the JWT secret.

## How do I test this?

1. Submit a grant application
2. Look at the links that get printed out in the console. Locally they use 3000 as the port. Change the port to 3001 and open the link in the browser. I couldn't figure out why we both run in 3000 and 3001.
3. Vote.

### CODE REVIEW

File: src/data/mutations/createSubmission.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  log('Sending grant review email', reviewer.email, submission.Id);
 
-  const submission = await Submission.findOne({
-    id: submissionId,
-  });
-
-  if (!submission) {
-    throw Error('Submission not found');
-  }
+  const expiresIn = 60 * 60 * 24 * 15; // 10 days
```
Comment (kristjanmik) [inline src/data/mutations/createSubmission.js]: The comment should say 15 days :P

**Review by kristjanmik — APPROVED** [review]
Looking good 🎉

File: src/data/mutations/createSubmission.js (hunk 1 above)
Comment (johannth) [reply src/data/mutations/createSubmission.js]: Lol. True.

Comment (johannth) [reply src/data/mutations/createSubmission.js]: Fixed in af27fe0b95b489af3f32c9d6ff0a6789c4a07dc0

**Pull request merged**